hw/drivers/flash/spiflash/include/spiflash/spiflash.h

```diff
@@ -44,6 +44,9 @@ struct spiflash_dev {
     const struct spiflash_chip *flash_chip;
 #if MYNEWT_VAL(OS_SCHEDULING)
     struct os_mutex lock;
+    struct os_callout apd_tmo_co;   /* Auto power down timeout callout */
+    os_time_t apd_tmo;              /* Auto power down timeout value (ticks) */
+    bool pd_active;                 /* Power down active */
```

Comment:
I could be beneficial to have pd_active visible even without scheduling. Then it could be used to automatically wake up flash on each access as it is done right now with this new feature.